docs/conventions.md

```diff
@@ -133,11 +172,20 @@ between monadic functions, and one for proofs between monadic functions and C
 definition that is instantiated to a `corres` / `ccorres` predicate by
 abbreviation.
 
- * General lemmas about `corres_underlying` that are directly used in program proofs are called `_corres` despite their more general nature. Exceptions are lemmas that are only used to build up the library internally, but are not used for program proofs.
+* General lemmas about `corres_underlying` that are directly used in program
+  proofs are called `_corres` despite their more general nature. Exceptions are
+  lemmas that are only used to build up the library internally, but are not used
+  for program proofs.
 
- * `corres_op` is for "built-in" operators and functions of the state monad such as `return`, `get`, `bind`, `when`, etc. Examples are `corres_get`, `corres_return`.
+* `corres_op` is for "built-in" operators and functions of the state monad such
+  as `return`, `get`, `bind`, `when`, etc. Examples are `corres_get`,
+  `corres_return`.
 
- * `function_corres` is for correspondence proofs between an abstract and a concrete function. Often these functions have the same name, the abstract in `underscore_style` the concrete in `CamelCase`. The lemma should use the concrete name for function (this is currently too random and should converge more).
+* `function_corres` is for correspondence proofs between an abstract and a
+  concrete function. Often these functions have the same name, the abstract in
+  `underscore_style` the concrete in `CamelCase`. The lemma should use the
+  concrete name for function (this is currently too random and should converge
+  more).
```

Comment:
Just commenting that it is probably a bit better than random now that we did some work on this, but it is not perfect for sure.

Reply:
Yes, that has improved. Should probably update this bit in a separate PR.